src/crud.py

```
@@ -1499,3 +1499,159 @@ async def get_duplicate_documents(
+                peer_id=peer_name,
+                session_id=session_id,
+                total_work_units=total,
+                completed_work_units=data["completed"],
+                in_progress_work_units=data["in_progress"],
+                pending_work_units=data["pending"],
+            )
+
+        return schemas.DeriverStatus(
+            sessions=sessions if sessions else None,
+            **base_response
+        )
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)